recipes/ginkgo/all/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.9)
```

Just our convention since there are other changes, trying to sneak these in.

```suggestion
cmake_minimum_required(VERSION 3.1)
```

---

Ginkgo cannot be built with CMake older than 3.9 if the CUDA option is enabled. Do you have any suggestion how to deal with this?

---

CMake does all that for us. if lower in your CMake tree you can set the minimum and it will be adjust accordingly...

This really is not important, and very cosmetic

---

Ah right thanks, I didn't realize that the check is executed at each level of CMakeLists.txt ;)